actions/views/lhs.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import Constants, {ActionTypes, SidebarSize} from 'utils/constants';
```

Comment:
Is there some specific reason behind this file being `js` instead of `ts`

Reply:
Files were mixed in the process of updating the branch. I deleted it now.